package.json

```diff
@@ -10,7 +10,7 @@
     "test": "npm run test-security",
```

Comment:
shouldn't "npm run lint" also generate the rules file, and shouldn't the gulpfile include the rules file in the things that it checks? and shouldn't the pre-commit file do that as well? thanks!